codegens/nodejs-request/lib/request.js

```diff
@@ -120,6 +145,13 @@ function getOptions () {
       type: 'boolean',
       default: false,
       description: 'Remove white space and additional lines that may affect the server\'s response'
+    },
+    {
+      name: 'Generate the code snippet with ES6 features',
+      id: 'ES6_enabled',
+      type: 'boolean',
+      default: false,
+      description: 'Allows the user to generate code snippet with latest EcmaScript 6 (ES6) features'
```

Comment:
This description is user-facing. So we don't need to specify 'user'. We have to come up with something that the user will read. 
We could go with something like: "Modifies code snippet to incorporate ES6 (EcmaScript) features."